wfe2/wfe_test.go

```diff
@@ -1097,23 +1098,18 @@ func TestChallenge(t *testing.T) {
 	}
 }
 
-// MockRAStrictUpdateAuthz is a mock RA that enforces authz status in `UpdateAuthorization`
+// MockRAUpdateAuthorizationError is a mock RA that just returns an error on UpdateAuthorization
```

Comment:
Ditto for the WFE2: This comment says `MockRAUpdateAuthorizationError` but the type is called `MockRAStrictUpdateAuthz`

Maybe you meant to rename the type throughout both `_test.go` files?